workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -683,6 +700,11 @@ public struct Request
 
             public static Request CreateRequest(Improbable.Worker.Query.EntityQuery entityQuery, uint? timeoutMillis = null, Object context = null)
             {
+                if (entityQuery.ResultType is SnapshotResultType)
+                {
+                    Debug.LogWarning("Cannot safely access component data from entity query - this is a known bug. To protect the integrity of this worker, this request will be dropped before sending");
```

Comment:
@jessicafalk @firtina-improbable is this warning message satisfactory?

Reply:
Is this a bug in the Core SDK or the GDK?
Do we know when it will be fixed?

Reply:
GDK and when we get around to it (after M2)

Reply:
I've cut a ticket to fix it properly post-M2

r.e. if it's Core SDK or GDK - @jamiebrynes7 would know better

Reply:
Let's add it to the known issues section (in this PR or the next)

Reply:
I've added this to the known issues 👍